internal/types/handlerenvironment.go

```diff
@@ -6,9 +6,15 @@ type HandlerEnvironment struct {
 	Version            float64 `json:"version"`
 	Name               string  `json:"name"`
 	HandlerEnvironment struct {
-		HeartbeatFile string `json:"heartbeatFile"`
-		StatusFolder  string `json:"statusFolder"`
-		ConfigFolder  string `json:"configFolder"`
-		LogFolder     string `json:"logFolder"`
+		HeartbeatFile       string `json:"heartbeatFile"`
```

Comment:
HandlerEnvironment.json  is supplied by Guest Agent, correct? Have the newly introduced values always been there? Does it need a particular version of the Guest Agent?  What happens when the newly added fields are absent from the HandlerEnvironment.json file?

Reply:
Not all of these have always been there - such as EventsFolder and EventsFolderPreview - but most have long histories. If a value isn't there, then it will remain at the default. I took these from the extensions platform - so if there was an issue I expect we would have seen it in VmApps.